addons/udes_stock/models/stock_picking_print_strategy.py

```diff
@@ -23,3 +23,11 @@ def strategies(self, picking):
         return self.search([
             ('picking_type_id', '=', picking.picking_type_id.id),
         ])
+
+    @api.multi
+    def records(self, picking, report):
```

Comment:
This changes the signature of the parent class and causes some test failures.

Reply:
Fixed